packages/desktop-client/src/components/reports/graphs/LineGraph.tsx

```diff
@@ -105,11 +109,21 @@ export function LineGraph({ style, graphData, compact }: LineGraphProps) {
                 {!compact && (
                   <>
                     <CartesianGrid strokeDasharray="3 3" />
-                    <XAxis dataKey="x" />
-                    <YAxis dataKey="y" tickFormatter={tickFormatter} />
+                    <XAxis dataKey="date" />
+                    <YAxis name="Value" tickFormatter={tickFormatter} />
                   </>
                 )}
-                <Line type="monotone" dataKey="y" stroke="#8884d8" />
+                {graphData.legend.map((legendItem, index) => {
```

Comment:
nit: can we match the existing labels in other graph files. I'd like to keep it consistent.

Reply:
Sure. I've also updated the names in BarLineGraph for completeness.